extensions/blocks/instagram-gallery/image-transition.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useState, useRef } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+
+export default function ImageTransition( { src } ) {
+	const [ loaded, setLoaded ] = useState( false );
+	const [ size, setSize ] = useState( 'auto' );
+
+	const img = useRef();
+
+	const transitionSpeed = '.5';
```

Comment:
Is there a reason why we are defining this as constant?
AFAICS we only use it one time. 🤔

Reply:
@pablinos suggested it above to make it easier to configure in the future :)

Reply:
it was used in a couple of places in a previous iteration - as it is only set in one place now have removed the constant. Would only make sense to make it a prop if we decide to abstract this component to shared one that can be used in other gallery contexts at any point.